tests/unit/transforms/fd-definition-test.js

```diff
@@ -1,12 +1,58 @@
+import Ember from 'ember';
 import { moduleFor, test } from 'ember-qunit';
+import FdViewAttributesProperty from 'ember-flexberry-designer/objects/fd-view-attributes-property';
+import FdViewAttributesMaster from 'ember-flexberry-designer/objects/fd-view-attributes-master';
+import FdViewAttributesDatail from 'ember-flexberry-designer/objects/fd-view-attributes-datail';
 
-moduleFor('transform:fd-definition', 'Unit | Transform | fd definition', {
-  // Specify the other units that are required for this test.
-  // needs: ['serializer:foo']
-});
+moduleFor('transform:fd-definition', 'Unit | Transform | fd definition');
 
-// Replace this with your real tests.
 test('it exists', function(assert) {
+
+  let objectModel = Ember.A([
+    FdViewAttributesProperty.create({
+      name: 'TestProperty',
+      caption: 'Test Property',
+      path: 'pathTestProperty',
+      visible: 'False',
```

Comment:
[In an object](https://github.com/Flexberry/ember-flexberry-designer/blob/develop/addon/objects/fd-view-attributes-property.js#L43), this property is defined as a boolean.